armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -374,6 +373,12 @@ def test_applyStateToOriginal(self):
             self.r.core.calcTotalParam("power", generationNum=2), totalPower
         )
 
+        self.converter.updateReactionRates()
+        for a in self.r.core:
+            for b in a:
+                self.assertTrue(b.p.rateAbs)
```

Comment:
Are the cross sections here 1.0, then? Otherwise I'm not seeing how this assertsTrue.